### ISSUE TITLE
ER: Wins randomizer content management

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#4282]
### Dependency
- #4190 - need the documentation that this issue delivers before we can start on this ER

### Emergent Requirement - Problem
We need a way to 
- know when we need specific practice area wins cards for the randomizer
- see at a glance which wins are in the randomizer (per practice area)
- keep track of which wins have been in the randomizer (in the past)

### Issue you discovered this emergent requirement in
- #2304

### Date discovered 
2023-03-24

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@ExperimentsInHonesty 

### What happens if this is not addressed
We are likely to end up with too many of one practice area in the randomizer  or a super manual process for auditing that.  And no plan for how to know what the past states were, etc.

### Resources
[wins spreadsheet]

### Recommended Action Items
- [ ] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Categories

- Community Management (we call it admin but we should move to calling it Community Management which is industry standard)
- Content
- Data
- Design
- Engineering
- Dev Ops
- PM (Project and Product)
- Research
- SEO/Marketing
- Fundraising

**Comment by roslynwythe** [comment]
Here is a suggestion for a solution:
![image](https://user-images.githubusercontent.com/5314153/228010256-0ac18626-8c26-4b1d-b298-856f9d3de9cb.png)
A wins entry will be displayed in the randomizer only if two conditions are met:
 - the "Display Homepage" column (sourced from the Review sheet) is TRUE
 - the appropriate "Enable Role on Homepage" column is checked. 

In response to a change in either of any of these cells, a function will recalculate the % views for each role and will document those in a row as shown above.

**Comment by roslynwythe** [comment]
- See #2505
